webui/internal/assets/package.json

```diff
@@ -8,31 +8,31 @@
     "cover": "istanbul cover _mocha --root src/ --include-all-sources -x '**/*.test.js' -x 'index.js' -- 'src/**/*.test.js' --require babel-register --require ignore-styles; exit 0"
   },
   "devDependencies": {
-    "babel-core": "^6.26.3",
-    "babel-eslint": "^10.1.0",
-    "babel-loader": "^8.0.6",
+    "babel-core": "latest",
```

Comment:
Why can't we lock this to recent most version? `latest` could install a package which might have breaking change for us.

Reply:
You have a good point. There are some GUI tests in the assets folder, I'll attempt to enable them on Travis, so we'll see whether it's breaking outright or not.